test/user-test.js

```diff
@@ -3,57 +3,137 @@ import { expect } from 'chai';
+
+    expect(user1.favoriteRecipes).to.deep.equal([recipeData[0], recipeData[1]]);
   });
 
-  it('Should be able to add recipes to favoriteRecipes', () =>{
-    user1.addToFavorites(recipeData[0])
-    expect(user1.favoriteRecipes.includes(recipeData[0])).to.eql(true);
+  it('Should be able to remove recipes from favoriteRecipes', function() {
+    user1.addToFavorites(recipeData[0]);
+    user1.addToFavorites(recipeData[1]);
+    user1.addToFavorites(recipeData[2]);
+    user1.removeFromFavorites(recipeData[1]);
+
+    expect(user1.favoriteRecipes).to.deep.equal([recipeData[0], recipeData[2]]);
+  });
+
+  it('Should be able to filter through favoriteRecipes by tag', function() {
+    user1.addToFavorites(recipeData[0]);
+    user1.addToFavorites(recipeData[1]);
+
+    expect(user1.filterFavorites('antipasti')).to.deep.equal([recipeData[0]]);
   });
 
-  it('Should be able to remove recipes from favoriteRecipes', () =>{
-    user1.removeFromFavorites(recipeData);
-    expect(user1.favoriteRecipes).to.eql([]);
+  it('Should be able to search favoriteRecipes ingredient', function(){
+    user1.addToFavorites(recipeData[0]);
+    user1.addToFavorites(recipeData[1]);
+
+    expect(user1.findFavorites('egg')).to.deep.equal([recipeData[0]]);
   });
 
-  it('Should be able to filter through favoriteRecipes by tag', () => {
+  it('Should be able to return multiple search of favoriteRecipes by ingredient'
+    , function(){
+      user1.addToFavorites(recipeData[0]);
+      user1.addToFavorites(recipeData[1]);
+      user1.addToFavorites(recipeData[5]);
+
+      expect(user1.findFavorites('egg')).to.deep.equal([recipeData[0],
+        recipeData[5]]);
+    });
+
+  it('Should be able to search favoriteRecipes by name', function(){
     user1.addToFavorites(recipeData[0]);
     user1.addToFavorites(recipeData[1]);
-    expect(user1.filterFavorites('antipasti')).to.eql([recipeData[0]]);
+
+    expect(user1.findFavorites('Chocolate')).to.deep.equal([recipeData[0]]);
   });
 
-  it('Should be able to search favoriteRecipes by name or ingredient', () => {
+  it('Should be able to search multiple favoriteRecipes by name', function(){
     user1.addToFavorites(recipeData[0]);
     user1.addToFavorites(recipeData[1]);
-    expect(user1.findFavorites('egg')).to.eql([recipeData[0]]);
+    user1.addToFavorites(recipeData[3]);
+    user1.addToFavorites(recipeData[6]);
+
+    expect(user1.findFavorites('Chocolate')).to.deep.equal([recipeData[0],
+      recipeData[6]]);
   });
 
-  it('Should be able to check ingredients in User/s pantry for a given recipe', () => {
+  it.skip('Should be able to check ingredients in User/s pantry for a given recipe', () => {
     expect(user1.checkPantry(recipeIngredients)).to.eql('You have the ingredients!');
   });
 
-  it('Should inform User if they lack required ingredients for a given recipe', () => {
+  it.skip('Should inform User if they lack required ingredients for a given recipe', () => {
```

Comment:
Here's the other one that needed to changed for consistency

Reply:
We can change it once we figure out which methods in these test will go to which class.